src/pybind/mgr/dashboard_v2/frontend/src/app/shared/datatable/table/table.component.scss

```diff
@@ -76,13 +76,48 @@
   }
   .input-group.dataTables_paginate {
     width: 8%;
+    min-width: 85px;
   }
 }
 
 ::ng-deep .oadatatable {
   border: $border-color;
   margin-bottom: 0;
   max-width: none!important;
+  .progress-linear {
+    display: block;
+    position: relative;
+    width: 100%;
+    height: 5px;
```

Comment:
Could  you use a mixin for
`
    display: block;
    position: relative;
    width: 100%;
    height: 5px;
`

Reply:
Not necessary after code cleanup.